src/ciflow-bot.ts

```diff
@@ -15,6 +15,7 @@ export class CIFlowBot {
   static readonly command_ciflow = 'ciflow';
   static readonly command_ciflow_rerun = 'rerun';
   static readonly allowed_commands: string[] = [CIFlowBot.command_ciflow];
+  static readonly allowed_repos = ['pytorch/pytorch'];
```

Comment:
Can we make this a bit more configurable in the future?

Ideally we should just check for the existence of a file within the repository so that we don't have to constantly update this variable every time we want to add a new repository to ciflow

Reply:
Yes, and that's a pretty good open-source path for ciflow, like having the repositories specify its own ciflow config, and turn ciflow-bot into a generic Github app. 

But currently ciflow-bot lives together with pytorch-probot, - i.e. sometimes even if the repository has the ciflow config file but didn't have the integration setting to connect to probot, it has no effect.